app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react'
+import Modal from 'react-bootstrap/lib/Modal'
+
+import DownloadCommand from './DownloadCommand'
+import DownloadSelectionTable, {
+  newSelectedBoxesState, getSelectedFileIds, getSelectedFileStats, bytesToSize
+} from './DownloadSelectionTable'
+import { fetchDownloadInfo } from 'lib/scp-api'
+
+
+/**
+  * a modal that, given a list of study accessions, allows a user to select/deselect
+  * studies and file types for download.  This queries the bulk_download/summary API method
+  * to retrieve the list of study details and available files
+  */
+export default function DownloadSelectionModal({ studyAccessions, show, setShow }) {
+  const [isLoading, setIsLoading] = useState(true)
+  const [downloadInfo, setDownloadInfo] = useState([])
+  const [selectedBoxes, setSelectedBoxes] = useState()
+  const [stepNum, setStepNum] = useState(1)
+
+  const { fileCount, fileSize } = getSelectedFileStats(downloadInfo, selectedBoxes, isLoading)
+  const prettyBytes = bytesToSize(fileSize)
+  const selectedFileIds = getSelectedFileIds(downloadInfo, selectedBoxes)
+
+  useEffect(() => {
+    if (show) {
+      setIsLoading(true)
+      fetchDownloadInfo(studyAccessions).then(result => {
+        setSelectedBoxes(newSelectedBoxesState(result))
+        setDownloadInfo(result)
+        setIsLoading(false)
+      })
+    }
+  }, [show, studyAccessions.join(',')])
+
+  let downloadButton = <button
+    className="btn btn-primary"
+    onClick={() => setStepNum(2)}
+    data-analytics-name="download-modal-next">
+    NEXT
+  </button>
+  if (fileCount === 0) {
+    downloadButton = <button className="btn btn-primary" disabled="disabled">
+      No files selected
+    </button>
+  }
+
+  return <Modal
+    id='bulk-download-modal'
+    className="full-height-modal"
+    show={show}
+    onHide={() => setShow(false)}
+    animation={false}
+    bsSize='large'>
+    <Modal.Body>
+      <div className="download-modal">
+        <div>
+          <div className="download-steps-header row">
+            <div className="col-md-4">
+              <h3 className={stepNum === 1 ? '' : 'greyed'}>
+                <span className="badge">1</span>
+                &nbsp; Select the files
+              </h3>
+            </div>
+            <div className="col-md-4">
+              <h3 className={stepNum === 2 ? '' : 'greyed'}>
+                <span className="badge">2</span>
+                &nbsp; Get terminal link
```

Comment:
Calling this a link is a bit confusing.  Users expect to click on links and go somewhere.  And while the command includes a link, but that's more of a technical detail.

```suggestion
                &nbsp; Get terminal command
```

I could also see labeling this "Get download command" or simply "Download".